zookeeper.service

```diff
@@ -4,9 +4,9 @@ After=network.target
 
 [Service]
 User=zookeeper
-Group=zookeeper
 EnvironmentFile=/etc/sysconfig/zookeeper
+Environment=CLASSPATH=@CLASSPATH@
```

Comment:
I suggest to change this to `CLASSPATH=.:/usr/share/java/zookeeper/*` and remove all CLASSPATH manipulations from the spec file